docs/uninstall/windows/python.rst

```diff
@@ -0,0 +1,26 @@
+Uninstalling Python
+=========================================
+
+
+.. note::
+
+   If you are running Python using Miniconda, you will have to :ref:`uninstall Miniconda <uninstall-conda>` to get rid of Python. 
```

Comment:
`get rid of` is probably something we shouldn't write. In this case ,simply remove that.